aya/src/programs/uprobe.rs

```diff
@@ -127,31 +160,30 @@ impl UProbe {
 fn resolve_attach_path<T: AsRef<Path>>(
     target: &T,
     pid: Option<pid_t>,
-) -> Result<Cow<'_, str>, UProbeError> {
+) -> Result<Cow<'_, Path>, UProbeError> {
```

Comment:
Do we still need Cow here? With the ldsocache changes to store PathBuf, 2/3  successful code paths yield a pathbuf, where the final one returns the input if the path is absolute.